OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -261,18 +262,24 @@ void SanityCheckPath(Mobile mobile)
 			return Pair.New(nextCell, subCell);
 		}
 
+		public override Color TargetLineColor
+		{
+			get
+			{
+				if (NextActivity != null)
+					return NextActivity.TargetLineColor;
+				return Color.Green;
+			}
+		}
+
 		public override IEnumerable<Target> GetTargets(Actor self)
 		{
-			if (path != null)
-				return Enumerable.Reverse(path).Select(c => Target.FromCell(self.World, c));
-			if (destination != null)
-				return new Target[] { Target.FromCell(self.World, destination.Value) };
-			return Target.None;
+			yield return Target.FromCell(self.World, this.destination.Value);
 		}
 
-		abstract class MovePart : Activity
+		public abstract class MovePart : Activity
```

Comment:
@forcecore Indeed, I made MovePart public in order to access it from the [RenderAboveShroud()](https://github.com/forcecore/OpenRA/blob/9948bb44dd0fa7852a1b57897a21b0f5d742d261/OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs#L70) method.

Great to hear this is beeing continued! :dolphin: